interface/src/app/about/about.tpl.html

```diff
@@ -1,8 +1,21 @@
-<div>
-  <h1 class="page-header">
-    About ADAGE
-  </h1>
-  <p>
-    This is where some explanatory text about ADAGE should go.
-  </p>
+<div class="page-header">
+  <h1>About ADAGE</h1>
 </div>
+<h3>
```

Comment:
I was using <p> initially, but the font size seems a little too small, and it looks bad with the <h1> header.